package-build/package-build.el

```diff
@@ -1030,6 +1030,24 @@ line per entry."
     (setq entries (sort entries (lambda (a b)
                                   (string< (symbol-name (car a))
                                            (symbol-name (car b))))))
+    (when (>= emacs-major-version 29)
+      (with-temp-file
+          (expand-file-name "elpa-packages.eld" package-build-archive-dir)
+        (let ((print-level nil)
+              (print-length nil))
+          (if pretty-print
+              (pp entries (current-buffer))
```

Comment:
Is it actually possible for this to produce something "prettier" than the below implementation?